configs/ansible-automation.json

```
@@ -0,0 +1,19 @@
+{
+    "roles": [
+      {
+        "name": "Ansible Automation Access",
+        "description": "An all access role which grants read and write permissions.",
+        "system": true,
+        "platform_default": true,
+        "version": 3,
```

Nitpick, but should this just be `2`?
```suggestion
        "version": 2,
```